lib/dateDisplay.ml

```diff
@@ -623,6 +623,57 @@ let short_marriage_date_text conf base fam p1 p2 =
     | None -> ""
   else ""
 
+(* ********************************************************************** *)
+(* [Fonc] short_family_date_text :
+            config -> base -> bool -> family -> string *)
+
+(* ********************************************************************** *)
+
+(** [Description] : Renvoie l'année marriage - séparation 
+                    ou uniquement l'année de la séparation.
+    [Args] :
+      - conf   : configuration de la base
+      - base   : base de donnée
+      - bool   : si faux, uniquement dates de la séparation
+      - family : famille
+    [Retour] : string
+    [Rem] : Exporté en clair hors de ce module.                           *)
+let short_family_dates_text conf _base marr_sep fam =
+  let marr_dates_aux =
+    match Date.od_of_cdate (Gwdb.get_marriage fam) with
+    | Some d -> (
+        match d with
+        | Dgreg (dmy, _) -> Some (prec_year_text conf dmy)
+        | _ -> Some "")
+    | _ -> Some ""
+  in
+  let sep_dates_aux =
+    match
+      List.find_opt
+        (fun e ->
+          e.efam_name = Efam_Divorce
+          || e.efam_name = Efam_Annulation
+          || e.efam_name = Efam_Separated)
+        (Gwdb.get_fevents fam)
+    with
+    | Some e -> (
+        match Date.od_of_cdate e.efam_date with
+        | Some d -> (
+            match d with
+            | Dgreg (dmy, _) -> Some (prec_year_text conf dmy)
+            | _ -> Some "")
+        | _ -> Some "")
+    | _ -> None
+  in
+  Adef.safe
+  @@
+  if marr_sep then
+    match (marr_dates_aux, sep_dates_aux) with
+    | Some m, Some s -> m ^ "-" ^ s
+    | Some m, None -> m
+    | _, _ -> ""
+  else match sep_dates_aux with Some m -> m | _ -> ""
```

Comment:
please avoid matching on `_ -> ...` and write ` None -> ...` instead, it makes it easier to change code later and is more readable